src/test/ui/imports/tool-mod-child.stderr

```diff
@@ -0,0 +1,9 @@
+error[E0433]: failed to resolve: maybe a missing crate `clippy`?
+  --> $DIR/tool-mod-child.rs:1:5
+   |
+LL | use clippy::a::b;
+   |     ^^^^^^ maybe a missing crate `clippy`?
```

Comment:
This is inconsistent with the error for `use clippy::a;`:

```
error[E0432]: unresolved import `clippy`
 --> src/lib.rs:1:5
  |
1 | use clippy::a;
  |     ^^^^^^ `clippy` is a tool module, not a module
```

How should I change this code to get that error?

Reply:
Interestingly, the `clippy::a::b` error is the same as the error for `rustdoc::a::b`.

Reply:
Right, they're both tool lints.

Reply:
Before this change `clippy::a` and `rustdoc::a` gave different errors, but now they're the same... weird.

Reply:
Rustdoc isn't marked as a tool module in resolve: https://rust-lang.zulipchat.com/#narrow/stream/131828-t-compiler/topic/question.20about.20tool.20module.20import.20errors